vars/AmazonLinux.yml

```diff
@@ -0,0 +1,18 @@
+---
+apache_service: httpd
+apache_daemon: httpd
+apache_daemon_path: /usr/sbin/
+apache_server_root: /etc/httpd
+apache_conf_path: /etc/httpd/conf.d
+
+apache_vhosts_version: "2.4"
+
+__apache_packages:
+  - httpd24
```

Comment:
I'd say (regarding your question in the PR summary) it's fine to default to 2.4, and then users could (if they wanted) override by setting `apache_packages` in their own playbook vars later.